specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingAccount.json

```diff
@@ -70,960 +119,750 @@
     },
     "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}": {
       "get": {
+        "tags": [
+          "BillingAccount"
+        ],
         "description": "Gets a billing account by its ID.",
         "externalDocs": {
           "url": "https://docs.microsoft.com/en-us/rest/api/billing/"
         },
         "operationId": "BillingAccounts_Get",
+        "produces": [
+          "application/json"
+        ],
         "parameters": [
           {
-            "$ref": "./types.json#/parameters/apiVersionParameter"
+            "$ref": "./types.json#/parameters/billingAccountName"
           },
           {
-            "$ref": "./types.json#/parameters/billingAccountName"
+            "in": "query",
+            "name": "includeDeletedBillingAccounts",
+            "description": "When true, results will include Billing Accounts that have been deleted.",
```

Comment:
nit: use singular wording in name + description

Reply:
Why singular here? I don't understand. This can return more than one right?

Reply:
I understand now. This is get by id